internal/config/uplift.go

```diff
@@ -47,6 +47,7 @@ type Uplift struct {
 	GitLab        *GitLab       `yaml:"gitlab" validate:"omitempty"`
 	Hooks         *Hooks        `yaml:"hooks" validate:"omitempty"`
 	Env           []string      `yaml:"env" validate:"dive,min=1"`
+	DirtyFiles    []string 		`yaml:dirtyFiles validate:"omitempty"`
```

Comment:
Can you please update the struct tags to the following `` `yaml:"dirtyFiles" validate:"omitempty,dive,file"` ``. This will ensure it is correctly validated by the new `check` command.

You might need to check this works. But my understanding is that it will ignore an empty slice and then once this config is defined, it will ensure any named file actually exists within the file system